client/src/pinboard.tsx

```diff
@@ -151,7 +153,7 @@ export const Pinboard: React.FC<PinboardProps> = ({
           flex-grow: 1;
         `}
       />
-      {initialItems.data && (
+      {activeTab === "chat" && initialItems.data && (
```

Comment:
minor and not really for this PR, but we should probably call this discussion rather than chat